test/bdd/bddtests_test.go

```diff
@@ -10,15 +10,16 @@ import (
 	"flag"
 	"fmt"
 	"os"
+	"os/exec"
 	"strconv"
 	"strings"
 	"testing"
 	"time"
 
 	"github.com/cucumber/godog"
+	"github.com/hyperledger/aries-framework-go/pkg/common/log"
 	authbddctx "github.com/trustbloc/hub-auth/test/bdd/pkg/context"
 
-	"github.com/trustbloc/kms/test/bdd/dockerutil"
```

Comment:
imo, using dockerutil is safer than os/exec

Reply:
here, it's quite hard to justify the whole bunch of code (`compose.go` and `docker.go` from `dockerutil` package) + dependency (`github.com/fsouza/go-dockerclient`, that is obsolete now as docker has official SDK) - while `dockerutil` still [uses os/exec](https://github.com/trustbloc/kms/blob/d5fce5f1ea9e492d4a76e9da6edc5c92eed3c597/test/bdd/dockerutil/compose.go#L114) for calling `docker-compose`

the only reason, as for me, might be setup with multiple docker-compose files, that, with changes in this PR, is not a case anymore

Reply:
This could be a long debate, but one could argue that calling a compromised `docker-compose` binary will cause a threat (if the machine was compromised already). It's ok for now since docker is required to build the projects (assuming docker binaries are legit on our machines).

Also take note that all other projects follow this old pattern. If we change it here, all other projects including afgo must be updated as well.

Reply:
but `dockerutil` will not help against compromised binary, it uses the same method - just calling `docker-compose`, here is the snippet (check it [here](https://github.com/trustbloc/kms/blob/d5fce5f1ea9e492d4a76e9da6edc5c92eed3c597/test/bdd/dockerutil/compose.go#L114)):
```go
cmd := exec.Command(dockerComposeCommand, cmdArgs...) //nolint:gosec // subprocess launched with variable
cmd.Dir = dir
```

the only difference is that now that call is closer to the place where it is actually used and not hidden behind `Composition` abstraction (this [one](https://github.com/trustbloc/kms/blob/d5fce5f1ea9e492d4a76e9da6edc5c92eed3c597/test/bdd/dockerutil/compose.go#L25))

Reply:
> take note that all other projects follow this old pattern

VCT doesn't follow it :-)

Reply:
ok, right. So it should be ok then. Thanks for the clarification.